### PR TITLE
Checkout screen styling with product cards and line item components

### PR DESCRIPTION
**Pull request by thumn**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR
- Styled product cards and line items in checkout screen
- Changed orientation for clerk app to landscape mode
- Restyled top bar with customer's name

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

## Relevant Links
N/A

### Online sources
[Super helpful video on how flexbox works](https://www.youtube.com/watch?v=K74l26pE4YA)

[//]: # "Optional - copy links to any tutorial or documentation that was useful to you when working on this PR"

### Related PRs
N/A

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

## How to review
Test by playing around with the products on the checkout screen and adding them/removing them from the cart! Let me know if there are any visual bugs.

[//]: # "The order in which to review files and what to expect when testing locally"

## Next steps
- Fix distribution of space between products and cart/rewards/checkout button
- Start restyling login and customer phone number screens to be compatible with landscape mode

[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"

## Tests Performed, Edge Cases
- Visual testing on checkout screen

[//]: # "Hopefully we will add a testing suite/CI soon, but until then note down the steps you took to test locally"

### Screenshots
image (1)

[//]: # "Add screenshots of expected behavior - GIFs if you're feeling fancy!"

CC: @anniero98 @wangannie

[//]: # "This tags in both Annies as a default. Feel free to change, or add on anyone who you should be in on the conversation."

### CODE REVIEW

File: components/ProductCartCard.js [hunk 1]
```diff
@@ -9,10 +16,44 @@ import { ProductBody, ProductCardContainer } from '../styled/checkout.js';
...
+          marginLeft: 24,
+          marginRight: 26
+        }}>
+        <LineItemDetail style={{ justifyContent: 'flex-start' }}>
+          {product.detail}
+        </LineItemDetail>
+        <LineItemPoints style={{ justifyContent: 'flex-end' }}>
+          {product.points * product.cartCount} pts
+        </LineItemPoints>
+      </View>
```
Comment (wangannie) [inline components/ProductCartCard.js]: General restructuring of each line in this component:
Make a styled component `LineItemRow` that can be reused for the 3 rows of this LineItem. Within each row, directly place base component text styles. Use flex to get the positioning of each row, then use flex in the LineItem container.

Check out this example here that Myles helped with: https://output.jsbin.com/zoyokezemu

File: styled/checkout.js [hunk 2]
```diff
@@ -65,44 +29,16 @@ export const LineItem = styled.View`
   height: 88px;
   background: ${Colors.lightest};
   border: 1px solid ${Colors.lighter};
-  border-top-width: 1px;
-  border-bottom-width: 1px;
   border-left-width: 0px;
   border-right-width: 0px;
+  padding-left: 24px;
+  padding-right: 26px;
+  padding: 13px 26px 11px 24px;
```
[the remark this answers is not in the document]
Comment (wangannie) [reply styled/checkout.js]: Can you make this `padding: 12px 24px 12px 24px;` just for the sake of pretty numbers and consistency? I get that I think you're going for compensating for the border but it's kind of funky like this.

File: styled/checkout.js [hunk 3]
```diff
@@ -65,44 +29,16 @@ export const LineItem = styled.View`
   height: 88px;
```
Comment (wangannie) [inline styled/checkout.js]: Sorry I might've originally suggested this, but if you remove this height, the Qty line wouldn't look so smushed.

File: components/ProductDisplayCard.js [hunk 4]
```diff
@@ -17,15 +19,12 @@ function ProductDisplayCard({ product }) {
         }}
         style={{
           width: 113,
-          height: 87,
-          marginLeft: 8,
-          marginRight: 8,
-          marginTop: 7
+          height: 87
```
Comment (wangannie) [inline components/ProductDisplayCard.js]: Add 8px margin-bottom to this image.

File: app.json [hunk 5]
```diff
@@ -4,13 +4,9 @@
...
     "sdkVersion": "35.0.0",
-    "platforms": [
-      "ios",
-      "android",
-      "web"
-    ],
+    "platforms": ["ios", "android", "web"],
     "version": "1.0.0",
-    "orientation": "portrait",
+    "orientation": "landscape",
```
Comment (anniero98) [inline app.json]: Didn't know this was controlled here, woah

File: styled/checkout.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (thumn) [reply styled/checkout.js]: oh I actually got these numbers from the Figma! but it makes sense why they were rounded like that

File: screens/CheckoutScreen.js [hunk 6]
```diff
@@ -31,9 +31,9 @@ export default class CheckoutScreen extends React.Component {
     const productsData = await loadProductsData();
 
     this.setState({
-      customer,
-      currentPoints: customer.points,
-      rewardsAvailable: customer.rewardsAvailable,
+      customer: customer,
+      rewardsAvailable: Math.floor(customer.rewardsAvailable),
+      fullProducts: productsData,
```
Comment (anniero98) [inline screens/CheckoutScreen.js]: Nit: doesn't look like you're using this anywhere?

File: app.json (hunk 5 above)
Comment (thumn) [reply app.json]: it seems to work on my phone so hopefully it's the same for tablets!!

File: screens/CheckoutScreen.js (hunk 6 above)
Comment (thumn) [reply screens/CheckoutScreen.js]: i'll remove it!
